Paclet/Kernel/HyperBloch.wl

```diff
@@ -1294,6 +1929,107 @@ If[OptionValue[PBCCluster],
 ]
 
 
+(* ::Subsection::Closed:: *)
+(*Construct Hamiltonians for clusters*)
+
+
+(* ::Subsubsection::Closed:: *)
+(*Reciprocal*)
+
+
+(* Note: not PBCCluster compatible, handled in the main function *)
```

Comment:
What does that mean?

Reply:
The (NonReciprocal)TBClusterHamiltonian function can not be used to evaluate the Hamiltonian in real space for PBC-clusters defined on a HCModelGraph or HCSupercellModelGraph. The reason being, this functionality can already be executed in the (NonReciprocal)AbelianBlochHamiltonian function through the option PBCCluster. 

A similar comment for finite flakes with disclination defects, then emphasizes that once the function for finite flakes with disclination defects is upgraded such that the flakes can be endowed with periodic boundary conditions, the (NonReciprocal)TBDisclinationClusterHamiltonian automatically enforces the appropriate boundary conditions, i.e., it is PBCCluster compatible.